### DISCUSSION TITLE
Important for Enterprise

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6480] [General]
Important for Enterprise
AMS lacks a critical feature essential to the broadcast industry. If a stream is not live, it should automatically run a looped video until the live stream resumes. This functionality is crucial and should be implemented as soon as possible. It warrants a General Meeting to discuss and explain the necessity of this feature in more detail.

**Comment by Mohit-3196** [comment]
Hi @brixxnatt,
Thank you for your suggestion. It's a good feature request and we can plan its implementation based on the priority.
Currently, I don't see many requests for such a feature and there can be some workaround solution to achieve this on the client side.

If it is something that is a must for you, we can plan for it depending on the opportunity size.

Thank you,
Mohit

**Comment by brixxnatt** [comment]
Yes its a must on our side  , also we are on enterpise plan ,  there is nothing much to do on client side , as we dont use html player , its all m3u8 files

**Reply by Mohit-3196** [reply]
Which player do you use for playing the m3u8 files?

**Reply by brixxnatt** [reply]
roku
